Description: Disable job server for autopkgtest.
 The Debian CI system kept showing regressions for using multiple make jobs:
 .
     FAIL stderr: make[2]: warning: -j3 forced in submake: resetting jobserver mode.
 .
 Perhaps this will disable the jobserver.
Author: Felix Lechner <felix.lechner@lease-up.com>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: wolfssl-5.6.6/configure.ac
===================================================================
--- wolfssl-5.6.6.orig/configure.ac
+++ wolfssl-5.6.6/configure.ac
@@ -9366,7 +9366,7 @@ AC_CONFIG_FILES([Makefile
 AC_CONFIG_FILES([scripts/unit.test],[chmod +x scripts/unit.test])
 
 AX_CREATE_GENERIC_CONFIG
-AX_AM_JOBSERVER([yes])
+AX_AM_JOBSERVER([no])
 
 # See Automake 9.4.1 Built Sources Example
 AC_DEFUN([AX_OUT_OF_TREE_FILE],[
